src/include/rbd/features.h

```diff
@@ -64,4 +69,6 @@
                                     RBD_FEATURE_FAST_DIFF      | \
                                     RBD_FEATURE_JOURNALING)
 
+#define RBD_FEATURES_INTERNAL          	(RBD_FEATURE_MIGRATING)
```

Comment:
We would need to ensure that the API prohibits users from directly setting the ```MIGRATING``` flag. Also, would should filter these out from journal events so that rbd-mirror doesn't propagate an odd flag state.